src/app/core/ui/ui/ui.component.scss

```
@@ -142,3 +142,8 @@ mat-sidenav-container {
 .justify-content-end {
   justify-content: end;
 }
+
+app-powered-by ::ng-deep .powered-by {
```

Stylelint (17.11.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)